hmcts/rpx-xui-webapp
# ngintegration enable codecept

* codecept_migration on latest master

* smoke test command with codecept fixed

* test updates

* a11y test enabled

* a11y test report fixed

* staff UI test fix

* test updates

* disabled ngIntegration tomimprovmenet stability

* global search test update

* enable xuiIntegration tests

* Test updated on Priority column display to jufge after R4

* ng Integration test update

* ngIntegration test updates

* ngIntegration test fixes

* test fix

* functional parallel execution thread count increased

* type error fix

* Added test scenario for invalid date error msg (#3006)

* caseDetailsCheck now validates all tabs (#3040)

* test to verify search filters are retained (#3042)

* CVE suppress

* Media viewer functionality (#3009)

* Test to open dummy document working

* Scenarios for bookmark and redaction

* Added scenario that tests comments feature on MV

* EUI 8472 buffer/codecept migration new (#3060)

* Updated changes to the back end

* Commmented out the old ways that the actions were being placed given the permissions

* commit changes to the permissions of the mock data in feature file

* formatting

* Fix the feature file to have passing tests

* Fix the feature file to have passing tests

* Fix the feature file to have passing tests

* EUI 8419 manage links testing (#3073)

* commit changes to the permissions of the mock data in feature file

* formatting

* Fix the feature file to have passing tests

* Fix the feature file to have passing tests

* Fix the feature file to have passing tests

* integration tests branch to buffer (#3063)

* 4/6 global search tests passing

* Added mock data for sherlock holmes

* Search page field validation is now passing

* Test

* Pagination test passing

* fixed search page and pagination tests

* 6/8 passing

* cleaning up tags

* taskActions.feature all tests passing

* EUI-8488 fix tests tab (#3081)

* fix tests tab

* Task -> 7 passing Find fix to "assigned to" on ui

* Task -> 7 passing Find fix to "assigned to" on ui

* in progress, users.js update to cw

* task tab test updated

* updated priority, tests passing

* formatting

* formatting

---------



* E UI 8423 will changes (#3097)

* EUI-8423 BOOKINGS

* Integration tests booking UI mock setup fixed

* Work access tests 6/7

* Dates updated all tests passing

---------




* add suppressions

* Pass buffer branch tests (#3104)

* updated codecept -> integration testing

* updated codecept -> functional testing

* updated functional tests to passing

* add @functional_enabled tag

* add @functional_enabled tag

* suppressions

* typo not running e2e's - fixed

* All work filters tests fix (#3107)

* fix failing tests

* suppressions

---------




---------

**Before creating a pull request make sure that:**


Please remove this line and everything above and fill the following sections:


### JIRA link (if applicable) ###



### Change description ###



**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```
